Lesson_10/l10_h01_regex.py

```diff
@@ -0,0 +1,26 @@
+"""
+HW:10[Volkov Anton].
+
+open input.txt file and find all small english letters
+that match such conditions:
+target small letter round exactly with three capital english
+letters on the left and on the right
+Match examples:
+sdTRYaUVKn  -> matches "a"
+NTRSjARTb   -> no match ( not exactly 3 capital letters on the left)
+zDFGbOPNq   -> matches "b"
+Print Output as : "Result: "<your_found_human_readable_string">
+"""
+import re
+
+# 1. Write regex pattern
+pattern = r'(?<![A-Z])[A-Z]{3}([a-z])[A-Z]{3}(?![A-Z])'
```

Comment:
You see is we know that something was before something after - better to place all this in certain regex blocks for great readability  example
`(?<=[^A-Z][A-Z]{3})[a-z](?=[A-Z]{3}[^A-Z])`
 or use verbose flag to leave regex comments. Your solution works too but from the first glance it's a little bit difficult to say what exactly we are trying to search